doc/release/release_0.14.rst

```diff
@@ -0,0 +1,273 @@
+  regionprops; available in ``skimage.measure.inertia_tensor``. (#2603)
+- Cycle-spinning function for approximating shift-invariance by averaging
+  results from a series of spatial shifts:
+  ``skimage.restoration.cycle_spin``. (#2647)
+- Haar-like feature: ``skimage.feature.haar_like_feature``,
+  ``skimage.feature.haar_like_feature_coord``,
+  ``skimage.feature.draw_haar_like_feature``. (#2848)
+- Data generation with random_shapes function:
+  ``skimage.draw.random_shapes``. (#277)
+- Subset of LFW (Labeled Faces in the Wild) database:
+  ``skimage.data.cbcl_face_database``. (#2905)
+- Fully reworked montage function (now with a better padding behavior):
+  ``skimage.util.montage``. (#2626)
+
+
+Improvements
+------------
+- ``VisuShrink`` method for ``skimage.restoration.denoise_wavelet``. (#2470)
+- ``skimage.transform.resize`` and ``skimage.transform.rescale`` have a new
+  ``anti_aliasing`` option to avoid aliasing artifacts when down-sampling
+  images. (#2802)
+- Support for multichannel images for ``skimage.feature.hog``. (#2870)
+- Non-local means denoising (``skimage.restoration.denoise_nl_means``) has
+  a new optional parameter, ``sigma``, that can be used to specify the noise
+  standard deviation. This enables noise-robust patch distance estimation. (#2890)
+- New ``alignment`` parameter in ``skimage.feature.plot_matches``. (#2955)
+
+
+API Changes
+-----------
+- ``skimage.util.montage.`` namespace has been removed, and
+  ``skimage.util.montage.montage2d`` function is now available as
+  ``skimage.util.montage2d``.
+- ``skimage.morphology.binary_erosion`` now uses ``True`` as border
+  value, and is now consistent with ``skimage.morphology.erosion``.
+
+
+Deprecations
+------------
+- ``skimage.util.montage2d`` is deprecated and will be removed in 0.15.
+  Use ``skimage.util.montage`` function instead.
+- ``skimage.novice`` is deprecated and will be removed in 0.16.
+- ``skimage.transform.resize`` and ``skimage.transform.rescale`` have a new
+  ``anti_aliasing`` option that avoids aliasing artifacts when down-sampling
+  images. This option will be enabled by default in 0.15.
+- ``regionprops`` will use row-column coordinates in 0.16. You can start
+  using them now with ``regionprops(..., coordinates='rc')``. You can silence
+  warning messages, and retain the old behavior, with
+  ``regionprops(..., coordinates='xy')``. However, that option will go away
+  in 0.16 and result in an error. This change has a number of consequences.
+  Specifically, the "orientation" region property will measure the
+  anticlockwise angle from a *vertical* line, i.e. from the vector (1, 0) in
+  row-column coordinates.
+- ``skimage.morphology.remove_small_holes`` ``min_size`` argument is deprecated
+  and will be removed in 0.16. Use ``area_threshold`` instead.
+
+
+Highlights
+----------
+This release is the result of 8 months of work.
+It contains the following 105 merged pull requests by 42 authors:
```

Comment:
@soupault I should also add that authors are different from committers. As much as I would love to keep the `42` here for nerdy reasons, the actual number should be `37`.

See https://github.com/scikit-image/scikit-image/pull/2927#issuecomment-391544350 for the more info and the most recent output.